src/lib.rs

```diff
@@ -410,54 +455,109 @@ pub fn run() -> Result<ExitStatus> {
     }
 
     let host_version_meta = rustc::version_meta()?;
+
     let cwd = std::env::current_dir()?;
-    if let Some(metadata) = cargo_metadata_with_args(None, Some(&args), &mut msg_info)? {
+    if let Some(metadata) = cargo_metadata_with_args(None, Some(&args), msg_info)? {
         let host = host_version_meta.host();
-        let toml = toml(&metadata, &mut msg_info)?;
+        let toml = toml(&metadata, msg_info)?;
         let config = Config::new(toml);
         let target = args
             .target
             .or_else(|| config.target(&target_list))
             .unwrap_or_else(|| Target::from(host.triple(), &target_list));
-        config.confusable_target(&target, &mut msg_info)?;
+        config.confusable_target(&target, msg_info)?;
 
-        let image_exists = match docker::image_name(&config, &target) {
-            Ok(_) => true,
+        // Get the image we're supposed to base all our next actions on.
+        // The image we actually run in might get changed with `target.{{TARGET}}.dockerfile` or `target.{{TARGET}}.pre-build`
```

Comment:
Same thing with the comment length, this just wraps too much in a default editor. Maybe 100 line length for comments?

Reply:
It seems like `rustfmt` supports this using `wrap_comments` and `comment_width`, but only on nightly sadly.